platform/darwin/src/MGLMapSnapshotter.h

```diff
@@ -139,6 +158,37 @@ MGL_EXPORT
 - (void)cancel;
 
 /**
+ The zoom level.
+ 
+ The default zoom level is 0. This overwrites the camera zoom level if set.
```

Comment:
For consistency, let’s match the language used in MGLMapSnapshotOptions documentation:

https://github.com/mapbox/mapbox-gl-native/blob/d913f4be79f2970c0ba95c297816632241eb374b/platform/darwin/src/MGLMapSnapshotter.h#L32-L35